### PR TITLE
Zero trap amp arm command

### PR DESCRIPTION
**Pull request by MidgleyM**

### CODE REVIEW

**Review by NolanCampbell2357 — APPROVED** [review]
Let's start putting TODO comments on constants we need to set to help us remember to set them so we do not break the robot.

File: robotbase/src/main/java/frc/robot/commands/trapAmpArm/TrapAmpArmZeroCommand.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  }
+
+  @Override
+  public void initialize() {
+    Robot.trapAmpArm.set(Constants.TRAP_AMP_ARM.ZERO_SPEED);
+  }
+
+  @Override
+  public boolean isFinished() {
+    return Robot.trapAmpArm.getMotorVelocity() <= Constants.TRAP_AMP_ARM.ZERO_SPEED_STOP_TOLERANCE;
```
Comment (NolanCampbell2357) [inline robotbase/src/main/java/frc/robot/commands/trapAmpArm/TrapAmpArmZeroCommand.java:20]: You need an end() block to set the rotations count to zero

Comment (NolanCampbell2357) [reply robotbase/src/main/java/frc/robot/commands/trapAmpArm/TrapAmpArmZeroCommand.java:20]: And stop the motors, that is also important.

**Review by NolanCampbell2357 — CHANGES_REQUESTED** [review]

File: robotbase/src/main/java/frc/robot/commands/trapAmpArm/TrapAmpArmZeroCommand.java (hunk 1 above)
Comment (MidgleyM) [reply robotbase/src/main/java/frc/robot/commands/trapAmpArm/TrapAmpArmZeroCommand.java:20]: i forgot about the end block completly

File: robotbase/src/main/java/frc/robot/commands/trapAmpArm/TrapAmpArmZeroCommand.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
+package frc.robot.commands.trapAmpArm;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.Constants;
+import frc.robot.Robot;
+
+public class TrapAmpArmZeroCommand extends Command {
```
Comment (NolanCampbell2357) [inline robotbase/src/main/java/frc/robot/commands/trapAmpArm/TrapAmpArmZeroCommand.java:7]: Change class name to no include `command` suffix

**Review by NolanCampbell2357 — CHANGES_REQUESTED** [review]

**Pull request merged**